lib/rabbit_feed/client.rb

```diff
@@ -160,6 +161,10 @@ def parse_options argv
           opts[:environment] = arg
         end
 
+        o.on '-x', '--prefix VAL', 'Prefix for rabbit queue routing key' do |arg|
```

Comment:
How about: `Prefix to append to the routing and queue naming allowing exclusive publishing/consuming.`? Open to suggestions here